homeassistant/components/websocket_api/commands.py

```diff
@@ -260,6 +267,84 @@ def handle_get_states(
     connection.send_message(response2)
 
 
+@callback
+@decorators.websocket_command(
+    {
+        vol.Required("type"): "subscribe_entities",
+        vol.Optional("entity_ids"): cv.entity_ids,
+    }
+)
+def handle_subscribe_entities(
+    hass: HomeAssistant, connection: ActiveConnection, msg: dict[str, Any]
+) -> None:
+    """Handle subscribe entities command."""
+    # Circular dep
+    # pylint: disable=import-outside-toplevel
+    from .permissions import SUBSCRIBE_ALLOWLIST
+
+    if "state_changed" not in SUBSCRIBE_ALLOWLIST and not connection.user.is_admin:
```

Comment:
Isn't `state_changed` always in `SUBSCRIBE_ALLOWLIST` ?

Reply:
https://github.com/home-assistant/core/pull/68026